File: EventApp.API/Models/Order.cs
using System;
using System.Collections.Generic;

namespace EventApp.API.Models
{
    public class Order
    {
        public int Id { get; set; }
        public int StatusId { get; set; }
        public Status Status { get; set; }
        public User User { get; set; }
        public int UserId { get; set; }
        public decimal TotalAmount { get; set; }
        public DateTime OrderDate { get; set; }
        public DateTime PaymentDate { get; set; }
        public ICollection<OrderTicket> OrderTickets { get; set; }
    }
}

File: EventApp-SPA/src/app/models/order.ts
import { User } from './user';
import { Status } from './status';
import { OrderTicket } from './orderTicket';

export interface Order {
    id: number;
    status: Status;
    user: User;
    totalAmount?: number;
    orderDate?: Date;
    paymentDate?: Date;
    orderTickets?: OrderTicket[];
}
